### DISCUSSION TITLE
GIt remote Issues

### DISCUSSION THREAD

**Discussion by dubsumm** [community/community#44929] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am trying to push a local repo to a github repo for the first time> after init my remote i got this error msg:
fatal: remote error: 
   is not a valid repository name
  Visit https://support.github.com/ for help

this is the name of the repo:
git@github.com:dubsumm/monsters_rolodex.git
this is the output when git remote -v is executed: 
![image](https://user-images.githubusercontent.com/102333319/213807908-46bb573d-3f27-4faa-8322-6269b5512602.png)

Any clue how to resole my issue

**Comment by airtower-luna** [comment, marked as answer]
The `git remote -v` output shows there's a tilde (`~`) at the end of the configured URL which doesn't belong there. Just set the correct URL with [`git remote set-url`](https://git-scm.com/docs/git-remote#Documentation/git-remote.txt-emset-urlem):

```
git remote set-url origin git@github.com:dubsumm/monsters_rolodex.git
```

**Reply by dubsumm** [reply]
This was the problem, and thank you so much! I copied directly from github link to so idk how that happened. Thanks again for your (0)(0)
